symbolic-base/src/ZkFold/Symbolic/Data/Bool.hs

```diff
@@ -65,8 +64,6 @@ deriving instance Show (c Par1) => Show (Bool c)
 instance {-# OVERLAPPING #-} (Eq a, MultiplicativeMonoid a) => Show (Bool (Interpreter a)) where
     show (fromBool -> x) = if x == one then "True" else "False"
 
-deriving newtype instance Symbolic c => SymbolicData (Bool c)
```

Comment:
Why did this instance have to be moved?

Reply:
Hmmm, it might not have to have been. I had a cycle at one point but not sure if this in fact solved it or something else.